packages/cms-cli/lib/prompts.js

```diff
@@ -63,17 +67,17 @@ const PORTAL_API_KEY = {
   message: 'Enter the API key for your portal:',
```

Comment:
Should we align this with "found under the DOMAIN column at https://app.hubspot.com/myaccounts-beta" and add something like:

>(found at https://app.hubspot.com/l/api-key)

cc @iamw11y @markhazlewood

Reply:
We should make sure these messages are consistent, for sure. Rather than tackling this piecemeal, should I start a reference doc that lists all of the command/response possibilities in the CLI, with both success and error responses? This would save us from having to comb through `.JS` files for all of the possibilities and could live in a similar space as the glossary that I'm putting together, where it would function as another internal reference to ensure that our docs and interfaces are informative and consistent.

Reply:
Are we confident about how https://app.hubspot.com/myaccounts-beta works when a user has a single portal?

Reply:
I haven't tested this case, is there a way to create a test user to confirm this works?

Reply:
This does work fine when a user is un one account, however, I do still think it is a little awkward given when you try and copy the HID you get taken into the account, and that it differs from what we say in https://designers.hubspot.com/guides/getting-started#quick_start

Reply:
I agree that this is not ideal, however, I'm not sure there is a simple solution at the moment. What about adding the ability to easily copy the HID in the account dropdown in the top right -- a small copy icon button perhaps?

Reply:
> it differs from what we say in https://designers.hubspot.com/guides/getting-started#quick_start

The downside to this link is that it doesn't cover how to switch portals to get the ID for a different portal if the user has several, which isn't super difficult, but could still lead to confusion. The "myaccounts-beta" link at least shows all portals and their associated portal IDs.